lib/providers/services/eos_service.dart

```diff
@@ -306,7 +306,7 @@ class EosService {
             ..actor = accountName
             ..permission = "active"
         ]
-        ..data = {"voter": accountName, "id": id, "amount": amount.abs()}
```

Comment:
I think it already fixed in master

Reply:
Yes it is, but this is a merging branch

Reply:
I mean - I merged master into this branch

Reply:
It's really weird it is here though... hmmmm

Let me try to make a good PR